bin/lint-staged.js

```diff
@@ -12,13 +12,6 @@ if (supportsColor) {
 process.on('SIGINT', () => {})
 
 const pkg = require('../package.json')
-require('please-upgrade-node')(
-  Object.assign({}, pkg, {
-    engines: {
-      node: '>=10.13.0', // First LTS release of 'Dubnium'
-    },
-  })
-)
```

Comment:
I moved the check to lib/index.js so that it also applies when running lint-staged via Node.js instead of cli.